user/models.py

```diff
@@ -1,5 +1,42 @@
+from django.contrib.auth.base_user import BaseUserManager
 from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.utils.translation import gettext as _
+
+
+class UserManager(BaseUserManager):
+    use_in_migrations = True
+
+    def _create_user(self, email, password, **extra_fields):
+        if not email:
+            raise ValueError("The given email and password")
+        email = self.normalize_email(email)
+        user = self.model(email=email, **extra_fields)
+        user.set_password(password)
+        user.save(using=self._db)
+        return user
+
+    def create_user(self, email, password=None, **exra_fields):
```

Comment:
There is a typo in your function parameters. You probably meant to write 'extra_fields' instead of 'exra_fields'. It's important to maintain the correct naming conventions to avoid confusion in your code.